pkg/authorization/apis/authorization/deprecated_fields.go

```diff
@@ -51,7 +51,7 @@ var (
 	groupsToResources = map[string][]string{
 		buildGroupName:       {"builds", "buildconfigs", "buildlogs", "buildconfigs/instantiate", "buildconfigs/instantiatebinary", "builds/log", "builds/clone", "buildconfigs/webhooks"},
 		imageGroupName:       {"imagestreams", "imagestreammappings", "imagestreamtags", "imagestreamimages", "imagestreamimports"},
-		deploymentGroupName:  {"deploymentconfigs", "generatedeploymentconfigs", "deploymentconfigrollbacks", "deploymentconfigs/log", "deploymentconfigs/scale"},
+		deploymentGroupName:  {"deploymentconfigs", "deploymentconfigrollbacks", "deploymentconfigs/log", "deploymentconfigs/scale"},
```

Comment:
We probably want to deprecate `deploymentconfigrollbacks` too in favor of `deploymentconfigs/rollback` but it's a separate issue.

Reply:
Is this tracked anywhere?